localization/english/culture/wc_basic_innovations_l_english.yml

```
@@ -0,0 +1,27 @@
+﻿l_english:
+
+ ##############
+ # Tribal Era #
+ ##############
+ 
+ ######################
+ # Early Medieval Era #
+ ###################### 
+ 
+ innovation_animal_war_training:0 "Animal War Training"
+ innovation_animal_war_training_desc:0 "Through skills taught to us my the best beastmasters, we can communicate, train and discipline the animals around us."
```

Fixed one word and switched two around.
```suggestion
 innovation_animal_war_training_desc:0 "Through skills taught to us by the best beastmasters, we can discipline, train and communicate with the animals around us."
```